Build fails on RN 0.47.1

Here's the build log for RN 0.47.1 with RNCli 2.0.1 ->
```
╭─ashish at ashish-Inspiron-5520 in ~/test/SoundTester using ‹2.4.1›
╰─○ react-native run-android
Scanning 560 folders for symlinks in /home/ashish/test/SoundTester/node_modules (7ms)
JS server already running.
Building and installing the app on the device (cd android && ./gradlew installDebug)...
Incremental java compilation is an incubating feature.
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:react-native-audio-toolkit:preBuild UP-TO-DATE
:react-native-audio-toolkit:preReleaseBuild UP-TO-DATE
:react-native-audio-toolkit:checkReleaseManifest
:react-native-audio-toolkit:preDebugAndroidTestBuild UP-TO-DATE
:react-native-audio-toolkit:preDebugBuild UP-TO-DATE
:react-native-audio-toolkit:preDebugUnitTestBuild UP-TO-DATE
:react-native-audio-toolkit:preReleaseUnitTestBuild UP-TO-DATE
:react-native-audio-toolkit:prepareComAndroidSupportAppcompatV72301Library
:react-native-audio-toolkit:prepareComAndroidSupportSupportV42301Library
:react-native-audio-toolkit:prepareComFacebookFbuiTextlayoutbuilderTextlayoutbuilder100Library
:react-native-audio-toolkit:prepareComFacebookFrescoDrawee101Library
:react-native-audio-toolkit:prepareComFacebookFrescoFbcore101Library
:react-native-audio-toolkit:prepareComFacebookFrescoFresco101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipeline101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipelineBase101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipelineOkhttp3101Library
:react-native-audio-toolkit:prepareComFacebookReactReactNative0471Library
:react-native-audio-toolkit:prepareComFacebookSoloaderSoloader010Library
:react-native-audio-toolkit:prepareOrgWebkitAndroidJscR174650Library
:react-native-audio-toolkit:prepareReleaseDependencies
:react-native-audio-toolkit:compileReleaseAidl
:react-native-audio-toolkit:compileReleaseNdk UP-TO-DATE
:react-native-audio-toolkit:compileLint
:react-native-audio-toolkit:copyReleaseLint UP-TO-DATE
:react-native-audio-toolkit:compileReleaseRenderscript
:react-native-audio-toolkit:generateReleaseBuildConfig
:react-native-audio-toolkit:generateReleaseResValues
:react-native-audio-toolkit:generateReleaseResources
:react-native-audio-toolkit:mergeReleaseResources
:react-native-audio-toolkit:processReleaseManifest
:react-native-audio-toolkit:processReleaseResources
:react-native-audio-toolkit:generateReleaseSources
:react-native-audio-toolkit:incrementalReleaseJavaCompilationSafeguard
:react-native-audio-toolkit:compileReleaseJavaWithJavac
:react-native-audio-toolkit:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/home/ashish/test/SoundTester/node_modules/react-native-audio-toolkit/android/lib/src/main/java/com/futurice/rctaudiotoolkit/AudioPackage.java:22: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-audio-toolkit:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-audio-toolkit:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 21.0 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/android-setup.html
```
Next, I reverted to RN 0.46.1 with RNCli 2.0.1.
Here's the build log when using RN 0.46.1 with RNCli 2.0.1 ->
```
Scanning 560 folders for symlinks in /home/ashish/test/SoundTester/node_modules (8ms)
JS server already running.
Building and installing the app on the device (cd android && ./gradlew installDebug)...
Incremental java compilation is an incubating feature.
:app:preBuild UP-TO-DATE
:app:preDebugBuild UP-TO-DATE
:app:checkDebugManifest
:app:preReleaseBuild UP-TO-DATE
:react-native-audio-toolkit:preBuild UP-TO-DATE
:react-native-audio-toolkit:preReleaseBuild UP-TO-DATE
:react-native-audio-toolkit:checkReleaseManifest
:react-native-audio-toolkit:preDebugAndroidTestBuild UP-TO-DATE
:react-native-audio-toolkit:preDebugBuild UP-TO-DATE
:react-native-audio-toolkit:preDebugUnitTestBuild UP-TO-DATE
:react-native-audio-toolkit:preReleaseUnitTestBuild UP-TO-DATE
:react-native-audio-toolkit:prepareComAndroidSupportAppcompatV72301Library
:react-native-audio-toolkit:prepareComAndroidSupportSupportV42301Library
:react-native-audio-toolkit:prepareComFacebookFbuiTextlayoutbuilderTextlayoutbuilder100Library
:react-native-audio-toolkit:prepareComFacebookFrescoDrawee101Library
:react-native-audio-toolkit:prepareComFacebookFrescoFbcore101Library
:react-native-audio-toolkit:prepareComFacebookFrescoFresco101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipeline101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipelineBase101Library
:react-native-audio-toolkit:prepareComFacebookFrescoImagepipelineOkhttp3101Library
:react-native-audio-toolkit:prepareComFacebookReactReactNative0461Library
:react-native-audio-toolkit:prepareComFacebookSoloaderSoloader010Library
:react-native-audio-toolkit:prepareOrgWebkitAndroidJscR174650Library
:react-native-audio-toolkit:prepareReleaseDependencies
:react-native-audio-toolkit:compileReleaseAidl
:react-native-audio-toolkit:compileReleaseNdk UP-TO-DATE
:react-native-audio-toolkit:compileLint
:react-native-audio-toolkit:copyReleaseLint UP-TO-DATE
:react-native-audio-toolkit:compileReleaseRenderscript
:react-native-audio-toolkit:generateReleaseBuildConfig
:react-native-audio-toolkit:generateReleaseResValues
:react-native-audio-toolkit:generateReleaseResources
:react-native-audio-toolkit:mergeReleaseResources
:react-native-audio-toolkit:processReleaseManifest
:react-native-audio-toolkit:processReleaseResources
:react-native-audio-toolkit:generateReleaseSources
:react-native-audio-toolkit:incrementalReleaseJavaCompilationSafeguard
:react-native-audio-toolkit:compileReleaseJavaWithJavac
:react-native-audio-toolkit:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
:react-native-audio-toolkit:extractReleaseAnnotations
:react-native-audio-toolkit:mergeReleaseShaders
:react-native-audio-toolkit:compileReleaseShaders
:react-native-audio-toolkit:generateReleaseAssets
:react-native-audio-toolkit:mergeReleaseAssets
:react-native-audio-toolkit:mergeReleaseProguardFiles
:react-native-audio-toolkit:packageReleaseRenderscript UP-TO-DATE
:react-native-audio-toolkit:packageReleaseResources
:react-native-audio-toolkit:processReleaseJavaRes UP-TO-DATE
:react-native-audio-toolkit:transformResourcesWithMergeJavaResForRelease
:react-native-audio-toolkit:transformClassesAndResourcesWithSyncLibJarsForRelease
:react-native-audio-toolkit:mergeReleaseJniLibFolders
:react-native-audio-toolkit:transformNative_libsWithMergeJniLibsForRelease
:react-native-audio-toolkit:transformNative_libsWithSyncJniLibsForRelease
:react-native-audio-toolkit:bundleRelease
:app:prepareComAndroidSupportAppcompatV72301Library
:app:prepareComAndroidSupportSupportV42301Library
:app:prepareComFacebookFbuiTextlayoutbuilderTextlayoutbuilder100Library
:app:prepareComFacebookFrescoDrawee101Library
:app:prepareComFacebookFrescoFbcore101Library
:app:prepareComFacebookFrescoFresco101Library
:app:prepareComFacebookFrescoImagepipeline101Library
:app:prepareComFacebookFrescoImagepipelineBase101Library
:app:prepareComFacebookFrescoImagepipelineOkhttp3101Library
:app:prepareComFacebookReactReactNative0461Library
:app:prepareComFacebookSoloaderSoloader010Library
:app:prepareOrgWebkitAndroidJscR174650Library
:app:prepareSoundTesterReactNativeAudioToolkitUnspecifiedLibrary
:app:prepareDebugDependencies
:app:compileDebugAidl
:app:compileDebugRenderscript
:app:generateDebugBuildConfig
:app:mergeDebugShaders
:app:compileDebugShaders
:app:generateDebugAssets
:app:mergeDebugAssets
:app:generateDebugResValues
:app:generateDebugResources
:app:mergeDebugResources
:app:bundleDebugJsAndAssets SKIPPED
:app:processDebugManifest
:app:processDebugResources
:app:generateDebugSources
:app:incrementalDebugJavaCompilationSafeguard
:app:compileDebugJavaWithJavac
:app:compileDebugJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
:app:compileDebugNdk UP-TO-DATE
:app:compileDebugSources
:app:transformClassesWithDexForDebug
Running dex in-process requires build tools 23.0.2.
For faster builds update this project to use the latest build tools.
:app:mergeDebugJniLibFolders
:app:transformNative_libsWithMergeJniLibsForDebug
:app:processDebugJavaRes UP-TO-DATE
:app:transformResourcesWithMergeJavaResForDebug
:app:validateSigningDebug
:app:packageDebug
:app:assembleDebug
:app:installDebug
Installing APK 'app-debug.apk' on 'Moto G (4) - 7.0' for app:debug
Installed on 1 device.

BUILD SUCCESSFUL

Total time: 53.423 secs
Running /home/ashish/Android/Sdk/platform-tools/adb -s ZY2238GHF4 reverse tcp:8081 tcp:8081
Starting the app on ZY2238GHF4 (/home/ashish/Android/Sdk/platform-tools/adb -s ZY2238GHF4 shell am start -n com.soundtester/com.soundtester.MainActivity)...
Starting: Intent { cmp=com.soundtester/.MainActivity }

```
As it can be inferred from the logs, I was successfully able to build and run the app using RN 0.46.1

--- a/android/lib/src/main/java/com/futurice/rctaudiotoolkit/AudioPackage.java
+++ b/android/lib/src/main/java/com/futurice/rctaudiotoolkit/AudioPackage.java
@@ -19,7 +19,7 @@ public class AudioPackage implements ReactPackage {
         return modules;
     }
 
-    @Override
+    // Deprecated in RN 0.47
     public List<Class<? extends JavaScriptModule>> createJSModules() {
         return Collections.emptyList();
     }

